lib/qb_integration/stock.rb

```diff
@@ -54,10 +54,12 @@ def attributes(amount, is_update = false)
     end
 
     def inventories
+      puts items
```

Comment:
I'd love to keep this, but maybe with a setting? So that if I set config[:please_output_stuff_for_I_am_going_mad] then it will output to the log?

Reply:
Ah - no that should have been removed hahaha